pkg/intcode/intcode.go

```diff
@@ -174,7 +140,7 @@ func (r *jumpTrue) run(p *program, arg1, arg2, arg3 int) bool {
 	return true
 }
 
-func (r *jumpFalse) run(p *program, arg1, arg2, arg3 int) bool {
+func (*jumpFalse) run(p *program, arg1, arg2, arg3 int) bool {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [parameter 'arg3' seems to be unused, consider removing or renaming it as _](https://app.codacy.com/manual/afarbos/aoc/pullRequest?prid=5498712)